kordac/Kordac.py

```diff
@@ -2,22 +2,35 @@
 import mdx_math
 from kordac.KordacExtension import KordacExtension
 
-class Kordac():
+class Kordac(object):
+    """A converter object for converting markdown
+    to HTML"""
 
-    def run(self, md_string):
-        self.heading = 'I am a heading'
+    def run(self, markdown_string):
```

Comment:
dislike html as that may more common than not be a module/namespace and prefer the variable names _html_tree_ when variable type is an object and _html_string_ when the variable type is a string.